recipes/pybufr-ecmwf/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+devtoolset-2-gcc-gfortran
```

Comment:
@adding this does not ensure the end user will have the libs. I'd rather if we stick with the packages available in conda.